<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Project</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="FileSystem" name="fileSystem">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item alignment="Qt::AlignRight">
       <widget class="ThemeChanger" name="themeChanger">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Dark theme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CodeEditor" name="codeEditor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_newFile"/>
    <addaction name="action_saveFile"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="action_runFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCompile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_newFile">
   <property name="text">
    <string>New file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_saveFile">
   <property name="text">
    <string>Save file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_runFile">
   <property name="text">
    <string>Run file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>CodeEditor.h</header>
  </customwidget>
  <customwidget>
   <class>FileSystem</class>
   <extends>QListView</extends>
   <header>FileSystem.h</header>
  </customwidget>
  <customwidget>
   <class>ThemeChanger</class>
   <extends>QRadioButton</extends>
   <header>ThemeChanger.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
